test/featurizations/GraphNodeFeaturization_tests.jl

```diff
@@ -17,6 +17,7 @@ using ChemistryFeaturization.Featurization
         fzn1 = GraphNodeFeaturization(features)
         fzn2 = GraphNodeFeaturization(fnames)
 
+        @test output_shape(fzn1) == 24
```

Comment:
could we also have a small comment here that explains why the output shape is 24? If we ever have to update these tests later on, it might make it easier.